headless/hostapd.conf

```diff
@@ -19,4 +19,10 @@ hw_mode=g
 channel=6
 macaddr_acl=0
 ignore_broadcast_ssid=0
-wpa=0
+wpa=2
+wpa_passphrase=#OpenAPS
```

Comment:
fixed passphrases or passwords are a bad idea. Passwords are created either automatically (using randomizing equipment) or by a human. Otherwise thse passwords will end up in a password database, see https://en.wikipedia.org/wiki/Default_password

Reply:
You missed the discussion at https://github.com/openaps/oref0/pull/716#issuecomment-337799249 - this is not intended as a security measure (since we allow wifi connections to unsecured networks, we don't trust the network), but rather just as a way to deter casual connections from people looking for Internet or poking around.